program/src/processor/stake_tokens.rs

```diff
@@ -12,21 +23,150 @@ use crate::instruction::accounts::{Context, StakeTokensAccounts};
 ///
 /// 0. `[w]` Stake config account
 /// 1. `[w]` Validator stake account
-///     * PDA seeds: ['stake', validator, config_account]
+///          (PDA seeds: ['stake', validator, config_account])
 /// 2. `[w]` Token Account
 /// 3. `[s]` Owner or delegate of the token account
-/// 4. `[]` Validator vote account
-/// 3. `[]` Stake Token Mint
-/// 4. `[]` Stake Token Vault, to hold all staked tokens.
-///   Must be the token account on the stake config account
-/// 5. `[]` Token program
-/// 6.. Extra accounts required for the transfer hook
+/// 4. `[ ]` Validator vote account
```

Comment:
Is this needed in the end since we can derive from the stake account data?

Reply:
Good point, I removed the account.